hypothesis-python/src/hypothesis/internal/conjecture/shrinker.py

```diff
@@ -365,14 +367,14 @@ def incorporate_test_data(self, data):
         """Takes a ConjectureData or Overrun object updates the current
         shrink_target if this data represents an improvement over it,
         returning True if it is."""
```

Comment:
Oh, I just noticed that removing the return value makes this docstring slightly inaccurate.